padreader.go

```diff
@@ -10,6 +10,10 @@ import (
 // PaddedSize takes size to the next power of two and then returns the number of
 // not-bit-padded bytes that would fit into a sector of that size.
 func PaddedSize(size uint64) abi.UnpaddedPieceSize {
+	if size <= 127 {
```

Comment:
An `UnpaddedPieceSize` smaller than 127 (actually not a multiple of) is invalid, so maybe we should test that and reject this case, but I might be missing context on the rationale of this PR.

Reply:
correct, we should either reject them (UnpaddedPieceSizes under 127) all or let them have same behavior.

for now, `PaddedSize(32) != PaddedSize(126)`, that's what this pr is for

Reply:
besides, this function here is mainly used when we have find a suitable UnpaddedPieceSize for a piece of data of any size, so i think given 127 is fine here

Reply:
Makes sense, thanks for the clarification @dtynn!